src/game_api/state.cpp

```diff
@@ -625,6 +626,11 @@ LiquidPhysicsEngine* State::get_correct_liquid_engine(ENT_TYPE liquid_type)
     return nullptr;
 }
 
+size_t State::get_location()
+{
+    return this->location;
+}

```

Comment:
Simple functions like this should be in header, so that compiler can optimize this call away